tests/Application/src/Resources/config/doctrine/Product.orm.xml

```diff
@@ -6,7 +6,7 @@
                                       http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd"
 >
 
-    <entity name="Tests\Setono\SyliusCalloutPlugin\Entity\Product" table="sylius_product">
+    <mapped-superclass name="Tests\Setono\SyliusCalloutPlugin\Entity\Product" table="sylius_product">
```

Comment:
I guess there's no reason for this change and I do believe it's more correct to use `entity` when you're in an app context